CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## v1.5.0
 
+- [security] updated version range of `decompress` to fix known security vulnerability [#8924](https://github.com/eclipse-theia/theia/pull/8294)
+  - with an updated version range, downstream applications have the possibility to:
+    - upgrade their `decompress` version (to benefit from the fix in `4.2.1`) by performing `yarn upgrade decompress` (which fixes the security flaw).
+    - use an earlier vulnerable version of `decompress` (`4.2.0`) for performance reasons by using `yarn resolutions`.
```

Comment:
I am not sure it's wise to provide information about how to work-around the security fix. From the PoV of a naive downstream product maintainer, it may look like we are saying that either choice is equally valid. 

Practically speaking we can't prevent downstream projects from doing what they want, but at least we should not provide the excuse that we told them to do so.

Reply:
So I would suggest to reformulate the entry so it looks less like a choice to be made downstream but something they ought to do. 
Additional info that we could provide:
- a mention that it will have a performance impact. You can link to this [message](https://github.com/eclipse-theia/theia/pull/7715#issuecomment-667434288) that contains a ballpark estimate. 
- a link to the security flaw information